StPicoEvent/StPicoBTofPidTraits.cxx

```diff
@@ -1,3 +1,13 @@
+/***************************************************************************
+ *
+ * $Id$
+ *
+ * Author: Xin Dong (xdong@lbl.gov)
```

Comment:
Are you sure it is really necessary? We know that many things have been copied from muDst so, I don't know... Also, maybe not in this file but in the other ones there are contributions from other people as well...

I think the repository does a good job by keeping track of everyone's contributions. People can visit the contributions page if they really want to know who did what or check the repositories history.

How about listing the contributors in one place, say, in README that is displayed on the main page?